### ISSUE TITLE
Show progress while loading

### ISSUE THREAD

**Issue by jackson-laskey** [unbounded-enterprise/sample-app#40]
When clicking through the explorer/my nfts menus there are a few moments where the page is loading and looks like the ss below:

![Image](https://user-images.githubusercontent.com/25757801/231537598-b44ed892-98c3-4b16-8f54-48a5d155a898.png)

Would be great if we could show some sort of feedback to the user that the page is loading the NFTs

Am open to whatever is easiest for developers to implement quickly, doesn't need to show a percentage or anything short term but just some sort of feedback that the page is loading

For example, a gif like this: 

![Image](https://user-images.githubusercontent.com/25757801/231537830-72aa351c-781d-434c-bb02-113526af4da1.gif)

**Comment by jackson-laskey** [comment]
@dstraffon added this, haven't merged to main. Looks fine but has a white background. Thoughts on using a version with a transparent background?

**Comment by dstraffon** [comment]
yeah, transparent would probably work best because our background color is #F9FAFC rather than pure white.

including a transparent gif (from google) that we can use for the time being, will probably want to upgrade this in the future to something that shows progress but think this is a fine placeholder



![Image](https://user-images.githubusercontent.com/25757801/232855528-d5c5bf14-f722-41a8-9328-2cddc437f2a7.gif)
